### ISSUE TITLE
i didn't drop it anywhere restricted. It's literally in my downloads folder, and I'm running it as administrator too. But it still doesn't work.  The program is running in the background as I'm typing this and it's not making any difference.

### ISSUE THREAD

**Issue by zalUoU** [valignatev/hitboxer#33]
i didn't drop it anywhere restricted. It's literally in my downloads folder, and I'm running it as administrator too. But it still doesn't work.  The program is running in the background as I'm typing this and it's not making any difference.

_Originally posted by @zalUoU in https://github.com/valignatev/socd/issues/29#issuecomment-2254659658_

**Comment by valignatev** [comment]
Hmm, do you have a config file near the exe?

**Comment by zalUoU** [comment]
there is, yes. I cross-checked the key mappings too and it checks out. I tried binding esc to a different key and that worked too. It's just my WASD that doesn't work for some reason.

**Comment by zalUoU** [comment]
Yes but it still doesn't work for some reason. I cross-checked the key mappings and even tried binding esc to a different key. It did register the key in the config file. It just doesn't work with my WASD for some reason

**Comment by valignatev** [comment]
Could you show me your config file contents? Also, what's the name of the config file?

**Comment by zalUoU** [comment]
If you're on discord, Please ping me there, I'll be able to provide you with the information there itself. @kxrthiklol is my ID. I'm out at the moment but I will surely get back to you once I get back to my place.

**Comment by valignatev** [comment]
I am, but I don't know how to dm someone I'm not in the same server with. My handle is `valigo`, I think my dms are opened for everyone.

**Comment by zalUoU** [comment]
Hey, I added you on discord. Your DMs aren't open, just add me back and we can have a chat there.

**Comment by zalUoU** [comment]
Hey! I texted you with the screenshots you asked me for. Can you check?

**Comment by Jremy21** [comment]
I had mine in a folder was able to get it to work after just dropping both files in my download folder and running as admin works great so far
